src/tactic/reassoc_axiom.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2019 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Simon Hudon
+
+Reformula category-theoretic axioms in a more associativity-friendly way
+-/
+
```

Comment:
Could we have a module-level comment here, that explains that you can either use `reassoc_axiom` as a command, or add the `reassoc` tag? It would be good to be able to decipher this without reading the file.